docs/index.md

```diff
@@ -29,7 +29,7 @@ Configure your project for use with UI5 Tooling.
 ❯ ui5 init
 Wrote ui5.yaml:
 
-specVersion: '3.0'
+specVersion: '3.1'
```

Comment:
that could be a double quote

Reply:
Actually not! This is the output of the `ui5 init` command, which apparently used single quotes